docs/Test-Strategy-Info.mdx

```diff
@@ -1,31 +1,182 @@
+
+#### As little as possible
+The rule of thumb is that we should avoid writing an end to end test if there is an alternative to validating the functionality. The alternative can be a combination of layers, for example, an integration test combined with an API test (which doesn’t exist right now) can be enough to reach the same confidence as an e2e test with less cost.
+
+### PUMA approach
+Adopting the [PUMA](https://confluence.dev.bbc.co.uk/display/podtest/PUMA) approach can provide a framework to decide if a proposed test genuinely meets the definition of an E2E. According to PUMA, an E2E test should:
+
+P - Prove core functionality
+
+U - be Understood by all
+
+M - Mandatory
+
+A - Automated
+
+
+### How should we automate?
+Patterns, anti-patterns and practices for E2E tests
+
+In this section, we will document some of the patterns we use and the best practices to write better E2E tests.
+
+##### Cypress best practices
+A good place to start is the Cypress best practices page: https://docs.cypress.io/guides/references/best-practices
+
+Custom commands for common functionalities
+We have many custom commands that make common tasks easier, these are located within [cypress/e2e/commands](https://github.com/bbc/simorgh/blob/latest/cypress/support/commands/application.js). It’s good to have a look at these to have an overview of what’s possible, and consider abstracting common functionality as a [cypress command](https://docs.cypress.io/api/cypress-api/custom-commands) when it makes sense.
+
+#### Do not ignore “flaky” tests
+
+
+> There is no such thing as a flaky test.
+> 
+> Any test is designed to provide insight into the functionality of the piece of code it exercises. As a result, when the test completes it provides information about how the code performed the actions detailed in the test.
+> 
+> If the test is “green” (i.e. it confirms the actual behaviour matches the expected behaviour), then we are provided with the knowledge that at that time the system behaved as defined by the test.
+> 
+> If the test is “red” (i.e. it confirms the actual behaviour did not match expectation), then we are provided with the knowledge that at that time the system did not behave as defined by the test.
+> 
+> If a test remains permanently “red”, then we either misunderstood the purpose of the code and so need to review the expectations of the checks in the test or we have uncovered unexpected behaviour in the system under test.
+> 
+> If a test remains permanently “green”, then we confirm current behaviour continues to meet the expectations of the test.
+> 
+> However, a test that flickers between the two states indicates something is not right and should not be ignored! I believe describing these as flaky makes it easy to ignore potential problems either in the functionality of the system under test or the test itself.
+> 
+> quote from  @sframpton
```

Comment:
This is not rendering as expected. I think we need to remove it from the > section
```suggestion
[Simon Frampton, Principal Test Engineer, BBC](@sframpton)
```